utilities/backupable/backupable_db_test.cc

```diff
@@ -38,6 +38,141 @@ namespace ROCKSDB_NAMESPACE {
 
 namespace {
 
+class DummyFileChecksumGen : public FileChecksumGenerator {
+ public:
+  explicit DummyFileChecksumGen(const FileChecksumGenContext& /*context*/) {
+    checksum_ = 0;
+  }
+
+  void Update(const char* data, size_t n) override {
+    Random32 rand32(n > 0 ? static_cast<uint32_t>(data[0]) : 0);
+    checksum_ = rand32.Uniform(std::numeric_limits<uint32_t>::max());
```

Comment:
This is really weird. Is the goal to get a fresh random value or a seeded pseudorandom value? It seems we're getting a seeded pseudorandom value that depends on how the data is broken up in calls to Update()! And there would be 1/2^32 chance of match anyway. (Inconsistent inconsistency?)

Reply:
Setting a fix seed should work here...